src/main/java/digital/slovensko/autogram/server/AssetsEndpoint.java

```diff
@@ -0,0 +1,65 @@
+package digital.slovensko.autogram.server;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import digital.slovensko.autogram.server.dto.ErrorResponse;
+import digital.slovensko.autogram.server.errors.InvalidUrlParamException;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class AssetsEndpoint implements HttpHandler {
+    private static final List<File> assets = new ArrayList<>();
```

Comment:
Načo potrebujeme List of Files, ak na riadku 49 aj tak znova čítame súbor cez getResource?

Načo celkovo treba tento list. Viem si predstaviť, že je to kvôli nejakej bezpečnosti, že pri spustení sa do listu hodí zoznam naozaj existujúcich súborov v assets/ a tým pádom sa potom requestom nedá získať nič iné, aj keby tam boli nejaké bodky a podobne. Je to tu preto?

Reply:
Stačil by nám teda iba list stringov.

Reply:
Áno, list som spravil presne kvôli bezpečnosti. Upravím na list stringov.